### DISCUSSION TITLE
Zeek local site policy

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#3238] [Unsupported Versions] [answered]
Can the Zeek local site policy be managed from the Manager node? I am trying to add `@load policy/protocols/conn/mac-logging` to each sensor's `/opt/so/conf/zeek/local.zeek` file, so MAC addresses can be logged in Zeek connection logs.

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/3178
